setup.py

```diff
@@ -54,14 +53,29 @@ def read_file(fname):
 static_requirements = []
 static_dir = os.path.dirname(os.path.realpath(kolibri_dist.__file__))
 
-install_requires = [str(ir.req) for ir in parse_requirements('./requirements.txt', session=False)]
+install_requires = [
+    'cherrypy==6.0.2',
+    'django-filter==0.13.0',
+    'django-js-reverse==0.7.2',
+    'six==1.10.0',
+    'djangorestframework==3.3.3',
+    'django==1.9.7',
+    'colorlog==2.6.0',
+    'docopt==0.6.2',
+    'django-mptt>=0.8.4',              # apparently you also need to have the dependency_links packages be included in install_requires
```

Comment:
Oooh... I see. But that wouldn't necessary when installing the static sdist, right? Just when building the sdist? How about when installing the non-static pre-built sdist? Could we just always include django-mptt as a static resource for now?

(If users pull in the released 0.8.4 package, things will break for them in weird ways)

I asked about release dates: https://github.com/django-mptt/django-mptt/pull/478